### PR TITLE
Fix #7731: OutputPanel: add loaded attribute

### LINKED ISSUE #7731: OutputPanel: add boolean attribute to enforce deferred load

### PR DESCRIPTION
**Pull request by jepsar**
Fix #7731

### CODE REVIEW

File: docs/11_0_0/components/outputpanel.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ styleClass | null | String | StyleClass of the HTML container element
 layout | block | String | Shortcut for the css display property, valid values are block(default) and inline.
 deferred | false | Boolean | Deferred mode loads the contents after page load to speed up page load.
 deferredMode | load | String | Defines deferred loading mode, valid values are "load" (after page load) and "visible" (once the panel is visible on scroll).
+loadNeeded | false | Boolean | Indicates deferred load is needed.
```
Comment (melloware) [inline docs/11_0_0/components/outputpanel.md]: Hmmm not sure about `loadNeeded` name.  Maybe `deferredCheck` or `deferredEnabled` or `deferOnAjax`?  I will let @tandraschko chime in.

**Review by melloware — CHANGES_REQUESTED** [review]

**Comment by tandraschko** [comment]
Maybe just "loaded"? Or ready?

**Comment by jepsar** [comment]
I think I prefer `loaded` in that case as the event is called `load`.

**Comment by tandraschko** [comment]
then +1 for loaded

**Pull request merged**